pkg/proxy/iptables/proxier.go

```diff
@@ -1359,6 +1359,7 @@ func (proxier *Proxier) syncProxyRules() {
 	if proxier.healthzServer != nil {
 		proxier.healthzServer.UpdateTimestamp()
 	}
+	metrics.SyncProxyRulesLastTimestamp.Set(time.Now().Unix())
```

Comment:
There is a handy `.SetToCurrentTime()` method.

Reply:
Ah, thanks!